src/components/HomeHub.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
-import GetFilmsButton from './GetFilmsButton';
-import UserProfileButton from './UserProfileButton';
+import ButtonGoToFilmList from './ButtonGoToFilmList';
+import ButtonGoToUserProfile from './ButtonGoToUserProfile';
 
 import {useTranslation} from 'react-i18next';
 import '../utils/i18n';
 
-const HomeScreenHeader = (): React.JSX.Element => {
+const HomeHub = (): React.JSX.Element => {
   const {t} = useTranslation();
   return (
     <View testID="HeaderContainer" style={styles.header}>
       <Text style={styles.headerText}>{t('title')}</Text>
-      <GetFilmsButton />
-      <UserProfileButton />
+      <ButtonGoToFilmList />
```

Comment:
DRY. If you need a button that takes you somewhere, use props and make it reusable. naming it simply `Button` or `ThemedButton` or conventional `GFButton` (GhibliFilmsButton)